runtime/compiler/aarch64/codegen/J9CodeGenerator.cpp

```diff
@@ -69,3 +70,25 @@ J9::ARM64::CodeGenerator::allocateRecompilationInfo()
    {
    return TR_ARM64Recompilation::allocate(self()->comp());
    }
+
+uint32_t encodeHelperBranchAndLink(TR::SymbolReference *symRef, uint8_t *cursor, TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Since this always takes a `TR::CodeGenerator` parameter and now lives in this file, would you consider making this a member function of the `J9::CodeGenerator` class instead?

Reply:
Turned the function `encodeHelperBranchAndLink()` to a member of `J9::ARM64::CodeGenerator`.